### ISSUE TITLE
Update upgrade procedure once Camunda Cloud automation is finished

### ISSUE THREAD

**Issue by npepinpe** [camunda-cloud/camunda-cloud-documentation#58]
**Description**

We're currently working on automating the Zeebe upgrade procedure for Camunda Cloud. Based on our results, we can then come up with a more streamlined solution for on-prem users, and after that we should update the upgrade guide to describe it.

**Comment by npepinpe** [comment]
Superseded by #221
